### DISCUSSION TITLE
Welcome to MagiskOnWSALocal Discussions!

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/MagiskOnWSALocal#130] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PeterNjeim** [comment]
I'm Peter Njeim. I used to have a fork of this repository to provide a way for people to download MagiskOnWSA without building it themselves, as well as providing Full OpenGApps support, however it has since been [deleted due to violation of GitHub policies](https://github.com/LSPosed/MagiskOnWSALocal/issues/4#issuecomment-1376508507). I also occasionally check on the issues in this repository and see if anyone needs help.
